pkg/reconciler/taskrun/resources/taskref.go

```diff
@@ -179,7 +179,7 @@ func readRuntimeObjectAsTask(ctx context.Context, obj runtime.Object, k8s kubern
 	case *v1beta1.ClusterTask:
 		return convertClusterTaskToTask(*obj), nil, nil
```

Comment:
wondering if we shall also verify here after converting into v1 Task?

Reply:
I don't know if we support verification policies for cluster tasks? Do we?